mrp_bom_structure_xlsx_level_1/readme/DESCRIPTION.md

```diff
@@ -0,0 +1,3 @@
+This module extends the functionality of the MRP capabilities of Odoo,
+and allows you to export to level 1 of the BoM structure to MS Excel
+.XLSX format.
```

Comment:
Perhaps this definition can be improved a little. What do you think about adding this?

> If you have a Bill of Materials (`A`) for which one of the Components also has a Bill of Materials (`B`). The report will only display the Components within the first level for the selected Bill of Materials (`A`), but not the Components from the Bill of Materials (`B`).